pkg/reconciler/ingress/lister.go

```diff
@@ -161,12 +158,16 @@ func (l *gatewayPodTargetLister) listGatewayTargetsPerPods(gateway *v1alpha3.Gat
 				l.logger.Infof("Skipping Subset %v because it doesn't contain a port %q", sub.Addresses, portName)
 				continue
 			}
-			portNumberStr := strconv.Itoa(int(portNumber))
-			url := fmt.Sprintf(urlTmpl, server.Port.Number)
+			target := status.ProbeTarget{
+				PodIPs: sets.NewString(),
```

Comment:
It's really annoying that you can't create a set with predefined capacity :|

Reply:
:|